app/components/SignUp/BillingInfo/BillingInfoForm.js

```diff
@@ -73,9 +72,11 @@ export class BillingInfoForm extends React.Component {
                 />
               </div>
               <div className="hxCol hxSpan-6 align-right">
-                <Submit
-                  classNames="btn-wide hxBtn hxPrimary"
+                <Button
                   label={t('common:actions.basic.next')}
+                  variant="primary"
+                  size="wide"
```

Comment:
do you mean to pass wide as a prop here?
```suggestion
                  size="wide"
                  wide
```